dipy/workflows/reconst.py

```diff
@@ -21,6 +21,165 @@
 from dipy.reconst.shm import CsaOdfModel
 from dipy.workflows.workflow import Workflow
 
+from dipy.reconst import mapmri
+
+class ReconstMAPMRIFlow(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return 'mapmri'
+
+    def run(self, data_file, data_bvecs, data_bvals, b0_threshold=0.0,
+            laplacian=True, positivity=True, out_rtop='rtop.nii.gz', out_lapnorm='lapnorm.nii.gz',
+            out_msd='msd.nii.gz', out_qiv='qiv.nii.gz', out_rtap='rtap.nii.gz',
+            out_rtpp='rtpp.nii.gz', out_ng='ng.nii.gz', out_perng='perng.nii.gz',
+            out_parng='parng.nii.gz', small_delta=0.0129, big_delta=0.0218, save_metrics=[],
+            out_dir='', laplacian_weighting=0.05):
+        """ Workflow for the app-dipy-mapmri on Brain-Life (www.brain-life.org). 
```

Comment:
Please remove the mention of brainlife. 

The correct description here is: "Workflow for fitting the MAPMRI model (with optional Laplacian regularization)."

It happens to also work on brainlife, but that's not the only place it could work.